DRODLib/Character.cpp

```diff
@@ -3467,6 +3493,12 @@ bool CCharacter::CanPushOntoOTileAt(UINT wX, UINT wY) const
 	return false;
 }
 
+//*****************************************************************************
+bool CCharacter::CanBeNPCBeethro() const
```

Comment:
The name is unclear to me.  Maybe "IsNPCBeethro" instead?

A comment on this method, explaining the significance of what an NPC being Beethro is, would be helpful.